### PR TITLE
Update versions.tf to require Terraform v1.3.x

### PR DESCRIPTION
**Pull request by OLeonardoRodrigues**
## Issue Number

#222 

## Checklist before requesting a review
- [x] The pr title can be used to describe what this pr did in `CHANGELOG.md` file
- [x] I have executed pre-commit on my machine
- [x] I have passed pr-check on my machine

Thanks for your cooperation!

### CODE REVIEW

File: versions.tf [hunk 1]
```diff
@@ -1,5 +1,5 @@
 terraform {
-  required_version = ">= 1.2"
+  required_version = "~> 1.3"
```
Comment (lonegunmanb) [inline versions.tf:2]: Would ">= 1.3" be better?

**Review by lonegunmanb — CHANGES_REQUESTED** [review]
Thanks @OLeonardoRodrigues for opening this pr! `"~> 1.3"` would pin the major version at `3`, could we change it to `">= 1.3"`? Thanks!

**Comment by OLeonardoRodrigues** [comment]
Sure I'll update it

**Comment by OLeonardoRodrigues** [comment]
So, it looks like there's stuff broken on the pre-commit, I'll try to work around it

**Comment by OLeonardoRodrigues** [comment]
@microsoft-github-policy-service agree

**Comment by lonegunmanb** [comment]
Hi @OLeonardoRodrigues thanks for the update, it looks like your modification on `unit-fixture` folder blocked CI pipeline to clone this branch, would you please revert the modification on this folder and try again? Thanks!

**Review by lonegunmanb — CHANGES_REQUESTED** [review]
Thanks @OLeonardoRodrigues for the update, we've passed acc tests but failed the pr-check job, would you please revert your modification on `unit-fixture` folder? Thanks!

**Comment by lonegunmanb** [comment]
Thanks @OLeonardoRodrigues for opening this pr, since we haven't heard from you for a while, I just replicated this pr and merged the change via #238. Thanks for your contribution, I'm closing it now.

**Pull request closed without merging**